beacon_node/network/src/sync/block_lookups/delayed_lookup.rs

```diff
@@ -66,15 +57,13 @@ pub fn spawn_delayed_lookup_service<T: BeaconChainTypes>(
             let mut interval = interval_at(interval_start, slot_duration);
             loop {
                 interval.tick().await;
-                while let Ok(msg) = delayed_lookups_recv.try_recv() {
-                    match msg {
-                        DelayedLookupMessage::MissingComponents(block_root) => {
-                            beacon_processor
-                                .send_delayed_lookup(block_root)
-                        }
-                    }
+                let Some(slot) =   beacon_chain.slot_clock.now_or_genesis() else {
```

Comment:
I think rustfmt must still be a bit confused about let-else

```suggestion
                let Some(slot) = beacon_chain.slot_clock.now_or_genesis() else {
```